src/app/Header/Header.jsx

```diff
@@ -1,20 +1,40 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import Box from '../../components/Box';
 import Container from '../../components/Container';
 import Flex from '../../components/Flex';
+import axios from '../../lib/axios';
+import getAuthToken from '../../utils/getAuthToken';
 import Logo from '../Logo';
 import MenuLeft from './components/MenuLeft';
 import MenuRight from './components/MenuRight';
 
-const Header = () => (
-  <Box as="header" bg="white" borderBottom="@1" borderColor="border">
-    <Container>
-      <Flex>
-        <MenuLeft />
-        <Logo />
-        <MenuRight />
-      </Flex>
-    </Container>
-  </Box>
-);
+const Header = () => {
+  const [tokenizedUser, setTokenizedUser] = useState();
+
+  useEffect(() => {
+    setTokenizedUser(getAuthToken());
+
+    const interceptor = axios.interceptors.response.use((response) => {
+      setTokenizedUser(getAuthToken());
+
+      return response;
+    });
+
+    return () => {
+      axios.interceptors.response.eject(interceptor);
+    };
+  }, []);
+
+  return (
+    <Box as="header" bg="white" borderBottom="@1" borderColor="border">
+      <Container>
+        <Flex>
+          <MenuLeft tokenizedUser={tokenizedUser} />
```

Comment:
```suggestion
          <MenuLeft user={tokenizedUser} />
```
对次一级的组件来讲他们不需要知道 tokenized 这个概念

Reply:
好的懂了！马上改！